package.json

```diff
@@ -19,7 +19,7 @@
     "url": "https://github.com/dojo/has.git"
   },
   "scripts": {
-    "prepublish": "grunt dist",
+    "prepublish": "grunt peerDepInstall dist",
```

Comment:
wonder whether we should actually add a `grunt prepublish` task, so we don't have to clutter this even more across all the repos in future?